examples/data-objects/codemirror/src/index.ts

```diff
@@ -34,7 +31,7 @@ class CodeMirrorFactory extends RuntimeFactoryHelper {
             registry,
             buildRuntimeRequestHandler(
                 defaultRouteRequestHandler(defaultComponentId),
-                innerRequestHandler,
+                rootDataStoreRequestHandler,
```

Comment:
Probably can just omit this - I don't think this example needs request support beyond the `defaultRouteRequestHandler`.

```suggestion
```